kinetic/distribution.yaml

```diff
@@ -16619,5 +16619,16 @@ repositories:
       url: https://github.com/stonier/zeroconf_msgs.git
       version: indigo
     status: maintained
+   jaka_ros_package:
```

Comment:
It looks like this has 3 spaces in front, when it should have 2. See the [CI failure here](https://travis-ci.org/ros/rosdistro/jobs/561599239)

```
expected <block end>, but found '<block mapping start>'
  in "<string>", line 16622, column 4:
       jaka_ros_package:
       ^
```

CI will complain about the repositories not being in alphabetical order. Please place the `jaka_ros_package` entry in alphabetical order rather than appending it to the end of the file.